metricbeat/module/kafka/broker/_meta/fields.yml

```
@@ -37,14 +43,26 @@
       description: The log flush rate
       type: float
     - name: topic.net.bytes_in
-      description: The incoming byte rate
+      description: The incoming byte rate per topic
       type: float
     - name: topic.net.bytes_out
-      description: The outgoing byte rate
+      description: The outgoing byte rate per topic
       type: float
     - name: topic.net.bytes_rejected
-      description: The rejected byte rate
+      description: The rejected byte rate per topic
       type: float
     - name: topic.messages_in
+      description: The incoming message rate per topic
+      type: float
+    - name: net.bytes_in
```

Following field conventions this should probably be:
```suggestion
    - name: net.in.bytes_per_sec
```

---

@sorantis is this ok from product perspective or we should strictly follow the naming provided in the initial [issue](https://github.com/elastic/beats/issues/13366#issue-486253665)?

---

there's no strict requirement on the naming. The field names were taken from the ER attached to the original issue. Do we have the mentioned naming convention somewhere?

---

You can find the conventions here: https://www.elastic.co/guide/en/beats/devguide/7.5/event-conventions.html